src/autolabel/transforms/serp_api.py

```diff
@@ -105,7 +105,9 @@ async def _apply(self, row: Dict[str, Any]) -> Dict[str, Any]:
                 logger.error(
                     f"Missing query column: {col} in row {row}",
                 )
-        query = self.query_template.format_map(defaultdict(str, row))
+        query = self.query_template.format_map(
```

Comment:
can we also handle the case when all input columns are empty?

Reply:
what does that mean? The default dict changes all empty columns to ""

Reply:
I think perhaps we don't make query if no columns present in row?